tests/console/docker.pm

```diff
@@ -34,6 +34,13 @@ sub run {
 
     install_docker_when_needed;
 
+    # Test docker client can communicate with docker daemon
+    assert_script_run('docker info');
+
+    # Check for bnc#1072367
+    my $docker_info = script_output('docker info');
+    record_soft_failure('bnc#1072367 -  docker engine doesnt have seccomp support') unless ($docker_info =~ m{seccomp/});
```

Comment:
Please avoid bnc# , Novell times are over :) Use boo# or bsc#. Also I think the whole block is a bit redundant and should be simplified to just

```perl
record_soft_failure('boo#1072367 - docker engine doesnt have seccomp support') unless (script_output('docker info') =~ m{seccomp/});
```

because `script_output` should also fail the test if the "docker info" call fails. See the option "proceed_on_failure" on http://open.qa/api/testapi/#_script_output . And I do not think you need to repeat the bug id in a comment and not define a docker_info variable when it is not reused. … And less blank lines in between :)

Reply:
That's a very good recommendation. Let me do that ;) thx 👍